pixel-point/bytebase-typescript-next
# feat: blog page

This pull request introduces a blog pages with categories. Without posts (you can only check its hero section).

Please review the changes in the commits, with the exception of [fef1d65](https://github.com/pixel-point/bytebase-typescript-next/pull/15/commits/fef1d65fa2ddc371f663fb80d9fc52d152c8a80e) and [296cd10](https://github.com/pixel-point/bytebase-typescript-next/pull/15/commits/296cd10cdc32e80835fd57dc2c55cf2bf289772b). These two commits contain markdown and image files that were exported from the existing website.

TODO:

- [ ] blog posts

Steps to check:

- open preview 
  - [blog](https://bytebase-typescript-next-git-blog-page-bytebase-pixelpoint.vercel.app/blog/), 
  - [category](https://bytebase-typescript-next-git-blog-page-bytebase-pixelpoint.vercel.app/blog/category/announcement/)
- make sure that everything looks and feels as expected
